functests/1_performance/rt-kernel.go

```diff
@@ -80,7 +80,7 @@ var _ = Describe("[performance]RT Kernel", func() {
 
 			return logs
 
-		}, 15*time.Minute, 30*time.Second).Should(ContainSubstring("PREEMPT RT"))
+		}, 15*time.Minute, 30*time.Second).Should(ContainSubstring(testutils.RTKernel))
```

Comment:
please document (commit message) when this changed: first version of distro + kernel (at least either, better both) which highlighted this issue.